Integrations/SplunkPy/SplunkPy.py

```diff
@@ -438,7 +445,24 @@ def fetch_incidents(service):
     if len(incidents) < FETCH_LIMIT:
         demisto.setLastRun({'time': now, 'offset': 0})
     else:
-        demisto.setLastRun({'time': lastRun, 'offset': search_offset + FETCH_LIMIT})
+        demisto.setLastRun({'time': last_run, 'offset': search_offset + FETCH_LIMIT})
+
+
+def parse_time_to_minutes():
+    """
+    Calculate how much time to fetch back in minutes
+    Returns (int): Time to fetch back in minutes
+    """
+    number_of_times, time_unit = FETCH_TIME.split(' ')
+    if str(number_of_times).isdigit():
+        number_of_times = int(number_of_times)
+    else:
+        return_error('Error: Invalid fetch time, need to be a positive integer.')
```

Comment:
'1 minute' is not a positive integer, let's rephrase

Reply:
done